### PR TITLE
docker: init

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,48 +1,64 @@
...
-  skip_cleanup: true
-  email: admin@inspirehep.net
-  api_key:
-    secure: OVIKb2O/kbHkuVS08xzW1uD1s5s8zJVIGgVK+g54ZJo8kdowtE1hXlWfqQMcNEOO6mPvknC2VysGMipOauosoaBPzY14TsOHv/9f+TlVdzrsPS9NBtxoBDheqCHLSc5fjl7I74jjxAZo+kl3MEHZ8VcPIg3AzzUnsm189cAu183sNJmGh3v4pZ5R6J8T7IxRd2yClBjN4n5RGySHUpy0mUtYVOkaOxZQ8xSLXSHpp9m6FWTljQ0sxxUudq2y3pe7SyuPEmo2iY+ww9buNZ+CIXvxALHDDm93YGAkHdC+KifEpsBLf0DjPXPvLPEVSHu5I3I8z2Pztc060DzDPYzaUBOwK8vRfZSd9vpjuSNYUoea2LPGx7a9ZBAv6BoIoqkfa2oyX44Jy6u0mr24rwEphm73517nCIB4JXs0Ti5nomZS57HsbGyBN2GQ2FS/iiVPOeTkyQ3HtbLx44bJS1Naa+JQAfKrfSRpG2iNQHRQOPSjvDPs7UpvQBKp7Qy03WuWkzQKDdPo819awvCRkOKRG1JQ087CgFhra9OuNbZYpi0CG7LcNmfrgDB1bBUfbK3IE9z//EOrMjtfOc3buJ6x+8zal4UcENicM7emwhNOZs6oe30jNWSTFJi0FvGUl88krtG/cTFeP0fTOJS7MUMCyhwB2r4pcuXtepXJmHICRPo=
-  tag: dev
-  on:
-    repo: inspirehep/record-editor
-    branch: dev-release
-    node: 6.9.1
+  - provider: npm
```
Comment (harunurhan) [inline .travis.yml]: you can remove those `npm` deployments.

Comment (drjova) [reply .travis.yml]: that's a good point

File: .travis.yml [hunk 2]
```diff
@@ -1,48 +1,41 @@
...
-  api_key:
-    secure: OVIKb2O/kbHkuVS08xzW1uD1s5s8zJVIGgVK+g54ZJo8kdowtE1hXlWfqQMcNEOO6mPvknC2VysGMipOauosoaBPzY14TsOHv/9f+TlVdzrsPS9NBtxoBDheqCHLSc5fjl7I74jjxAZo+kl3MEHZ8VcPIg3AzzUnsm189cAu183sNJmGh3v4pZ5R6J8T7IxRd2yClBjN4n5RGySHUpy0mUtYVOkaOxZQ8xSLXSHpp9m6FWTljQ0sxxUudq2y3pe7SyuPEmo2iY+ww9buNZ+CIXvxALHDDm93YGAkHdC+KifEpsBLf0DjPXPvLPEVSHu5I3I8z2Pztc060DzDPYzaUBOwK8vRfZSd9vpjuSNYUoea2LPGx7a9ZBAv6BoIoqkfa2oyX44Jy6u0mr24rwEphm73517nCIB4JXs0Ti5nomZS57HsbGyBN2GQ2FS/iiVPOeTkyQ3HtbLx44bJS1Naa+JQAfKrfSRpG2iNQHRQOPSjvDPs7UpvQBKp7Qy03WuWkzQKDdPo819awvCRkOKRG1JQ087CgFhra9OuNbZYpi0CG7LcNmfrgDB1bBUfbK3IE9z//EOrMjtfOc3buJ6x+8zal4UcENicM7emwhNOZs6oe30jNWSTFJi0FvGUl88krtG/cTFeP0fTOJS7MUMCyhwB2r4pcuXtepXJmHICRPo=
-  tag: dev
-  on:
-    repo: inspirehep/record-editor
-    branch: dev-release
-    node: 6.9.1
+  - provider: script
+    skip_cleanup: true
+    script: curl -sL https://sentry.io/get-cli/ | bash; cd $TRAVIS_BUILD_DIR; ./build-and-deploy.sh
```
Comment (harunurhan) [inline .travis.yml]: just curios... why `curl sentry-cli`?

Comment (drjova) [reply .travis.yml]: left over actually good catch thanks

**Pull request merged**